src/main/java/bio/terra/service/iam/sam/SamIam.java

```diff
@@ -583,4 +584,18 @@ public RepositoryStatusModelSystems samStatus() {
         }
     }
 
+    @VisibleForTesting
+    static String extractErrorMessage(final ErrorReport errorReport) {
+        List<String> causes = new ArrayList<>();
+        for (ErrorReport cause: errorReport.getCauses()) {
+            causes.add(extractErrorMessage(cause));
+        }
+
+        String separator = (causes.isEmpty() ? "" : ": ");
+        String openParen = (causes.size() > 1 ? "(" : "");
+        String closeParen = (causes.size() > 1 ? ")" : "");
+        return errorReport.getMessage() + separator +
+            openParen + StringUtils.join(causes, ", ") + closeParen;
```

Comment:
No need to use `StringUtils` if the parameter is a String collection.
```suggestion
            openParen + String.join(", ", causes) + closeParen;
```

Reply:
Ah!  I was looking for this method.  Will update